actors/market/src/testing.rs

```diff
@@ -324,9 +322,17 @@ pub fn check_state_invariants<BS: Blockstore>(
             lock_table_count,
             deal_op_epoch_count,
             deal_op_count,
-            claim_id_to_deal_id,
             alloc_id_to_deal_id,
         },
         acc,
     )
 }
+
+/// Compute a deal CID directly (the actor code uses a runtime built-in).
+pub(crate) fn deal_cid(proposal: &DealProposal) -> Result<Cid, ActorError> {
```

Comment:
this is the only `pub(crate)` in here, do you happen to know why the other `pub`s aren't also limited? probably good practice.

Reply:
We essentially never use `pub(crate)`, I view this one as an accident (probably automated refactor).